canvas_modules/common-canvas/__tests__/test_resources/paramDefs/structurelisteditor_paramDef.json

```diff
@@ -1216,7 +1216,7 @@
       },
       {
         "complex_type_ref": "inlineEditingTableError",
-        "row_selection": "single",
+        "row_selection": "multiple-edit",
```

Comment:
This change is not needed. You can revert this to "single".

Reply:
I switched from single to multi select because single tables no longer have the toolbar which means that single tables can no longer move rows. The table has moveable rows set to true. I discussed this with Matt and Jesus, and they mentioned to switch it to multi select to ensure test coverage.